ui/components/app/name/name.tsx

```diff
@@ -83,7 +83,8 @@ export default function Name({
       <div
         className={classnames({
           name: true,
-          name__saved: hasName,
+          name__saved: hasPetname,
+          name__recognized_unsaved: !hasPetname && hasName,
```

Comment:
Would it make this logic more explicit if we rename `hasName` to `hasDisplayName`?

Reply:
Good catch! Done.